src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -290,7 +324,11 @@ export class MgtPersonCard extends MgtTemplatedComponent {
 
     // Phone
     let phone: TemplateResult;
```

Comment:
This is a great improvement, can you also update it below in the `renderAdditionalDetails` function? The phone doesn't show up when expanded otherwise

Reply:
Did you mean `renderContactDetails?`

Reply:
Agh, yes, that's right, I meant `renderContactDetails`